src/ui/dfn-list.js

```diff
@@ -78,7 +78,9 @@ function labelDfnIfExported(dfn) {
  * @param {HTMLElement} dfn a definition
  */
 function labelDfnIfUnused(dfn) {
-  const isUsed = document.querySelector(`a[href^="#${dfn.id}"]`);
+  const isUsed = document.querySelector(
+    `:not(.dfn-panel) > :not(li) > a[href^="#${dfn.id}"]`
```

Comment:
I'm not sure how this works, but it works :sweat_smile:

Reply:
Maybe add this 👆 as a comment? The query is indeed complex.